crates/assistant/src/context.rs

```diff
@@ -2099,10 +2117,17 @@ impl Context {
                 continue;
             }
 
+            let content = match request_type {
+                RequestType::Chat => Vec::new(),
+                RequestType::SuggestEdits => {
+                    vec![MessageContent::Text(SUGGEST_EDITS_PREAMBLE.to_string())]
+                }
+            };
+
```

Comment:
Because we're inside the `for message in self.messages(cx)` loop here, is this content going to be added to every message?

Reply:
Oh good catch! Yeah this isn't noticeable with a single message, but would be if there are more than one. I'll move it.

Reply:
Fixed in 3177f6b89165890c31c52ada77701bc3e8f7a5b8